spec/fixtures/fake_app.rb

```diff
@@ -30,7 +30,7 @@ class FakeApp < Sinatra::Base
 
       %i[get put post delete].each do |meth|
         send(meth, '/redirected') do
-          additional_info = meth == :get ? ", session #{session}" : " using #{meth} with #{params}"
+          additional_info = meth == :get ? ", session #{session}#{request.session_options}" : " using #{meth} with #{params}"
```

Comment:
If you change
    
`", session #{session}#{request.session_options}"`

to this instead, I think it would be more readable code:

`", session #{session} with options #{request.session_options}"`